### PR TITLE
Packge wrapping resist

### LINKED ISSUE #6835: Gift wrap lacks a resist
You can gift wrap players and they cannot break out by yourself, making it a strait jacket that is much faster to apply and comes in packs of 10.

### PR DESCRIPTION
**Pull request by clusterfack**
You can resist out of package wrap and its syndicate variety (which takes a longer timer)
Fixes #6835

### CODE REVIEW

**Comment by Exxion** [comment]
:+1:

**Comment by Intigracy** [comment]
Here's your imaginary dollar:

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -734,13 +734,27 @@ Thanks.
...
 			var/mob/living/Location = H.loc
 			Location.drop_from_inventory(H)
 		qdel(H)
 		H = null
 		return
+	else if(istype(src.loc, /obj/structure/strange_present))
+		var/obj/structure/strange_present/present = src.loc
+		forceMove(get_turf(src))
+		qdel(present)
+		return
```
Comment (Intigracy) [inline code/modules/mob/living/living.dm]: No break out message?

Comment (clusterfack) [inline code/modules/mob/living/living.dm]: It's instant, I think it'll be pretty obvious.

**Comment by Intigracy** [comment]
Not really if you put them to sleep then wrap them.

It'd be totally sneaky and you might not even notice them escaping.

**Comment by clusterfack** [comment]
Can you even resist while asleep? I admit there isn't any specific code here to prevent it.

**Comment by Intigracy** [comment]
You can when you wake up, I mean.

Actually this gives me an idea, are there any sounds that would fit with tearing out of paper that you could put in?

**Comment by Intigracy** [comment]
poster_ripped.ogg for example

**Comment by clusterfack** [comment]
Might be nice

**Comment by Probe1** [comment]
I tried to stare at the code until I figured out how to do this and failed.  Thank you for doing this senpai.

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -734,13 +734,29 @@ Thanks.
...
 		var/obj/item/weapon/holder/H = src.loc
-		src.loc = get_turf(src.loc)
+		forceMove(get_turf(src))
 		if(istype(H.loc, /mob/living))
 			var/mob/living/Location = H.loc
 			Location.drop_from_inventory(H)
 		qdel(H)
 		H = null
 		return
+	else if(istype(src.loc, /obj/structure/strange_present))
```
Comment (PJB3005) [inline code/modules/mob/living/living.dm:744]: Wow clusterfack I didn't review this because:
- I'm lazy.
- I assumed you knew better than this...

Comment (Exxion) [inline code/modules/mob/living/living.dm:744]: I'd be more worried about the possibility of vanishing items due to only the mob being moved out, ignoring anything the mob may have dropped

Comment (clusterfack) [inline code/modules/mob/living/living.dm]: If you had a problem with how this works you should have done it instead. It was clear nobody was about to.

**Pull request merged**